src/components/Event.js

```diff
@@ -6,9 +6,10 @@ class Event extends Component {
   }
 
   render() {
+    let className = this.props.currentEvent === this.props.id ? 'center event currentEvent' : 'event';
```

Comment:
Good idea to put the class name inside the code to dynamically set it based on whether it's the current event.